src/main/java/net/dries007/tfc/common/blocks/TFCBlocks.java

```diff
@@ -241,6 +238,19 @@ public final class TFCBlocks
         )
     );
 
+    public static final Map<Wood, Map<Metal.Default, Map<Class<? extends ITFCHangingSignBlock>, RegistryObject<SignBlock>>>> HANGING_SIGNS = Helpers.mapOfKeys(Wood.class, wood -> {
+        Supplier<ExtendedProperties> woodProps = () -> Wood.BlockType.properties(wood).noCollission().strength(1F).flammableLikePlanks().blockEntity(TFCBlockEntities.HANGING_SIGN).ticks(SignBlockEntity::tick);
+        WoodType woodType = wood.getVanillaWoodType();
+        return Helpers.mapOfKeys(Metal.Default.class, metal -> metal.metalTier() != Metal.Tier.TIER_0, metal -> {
```

Comment:
there's a helper method called like Metal#has that does this check. Check what 'Part' type chains have (i think its utility?) and check against that, don't use tier which is only for behavior